app/models/manageiq/providers/openstack/cloud_manager/cloud_volume_snapshot.rb

```diff
@@ -1,2 +1,138 @@
 class ManageIQ::Providers::Openstack::CloudManager::CloudVolumeSnapshot < ::CloudVolumeSnapshot
+  include SupportsFeatureMixin
+
+  supports :create
+  supports :update
+  supports :delete
```

Comment:
probably you want to skip the `:create` feature here and add `:create_volume_snapshot` to the base class of `cloud_volume.rb`